### ISSUE TITLE
Keyboard keystroke to enter date picker is not discoverable

### ISSUE THREAD

**Issue by backwardok** [airbnb/react-dates#1009]
For a keyboard user who doesn't use a screen reader, it's not clear how to access the calendar of the date picker. Even if you figure out how to open the keyboard shortcut menu, it doesn't explain how to get into the calendar.

For a screen reader user, there are instructions to use the down arrow key to access the calendar. However, there are a few problems with this. When using a screen reader, the down key (and the arrow keys in general) have special meaning and captured by the screen reader and not passed down to the UI. Additionally, when using a touch device without a keyboard, that action isn't even possible, making the instructions confusing.

**Comment by erin-doyle** [comment]
So perhaps things have changed since this was opened but at present, you can use tab to move from the input to the calendar.  I'm not sure if this is sufficient or not but at least that's now an option in addition to the down arrow key.  I really don't know what it should be though.  What would a keyboard only user expect?  Probably tab is acceptable, I'm guessing that would be the first or second thing they would try?  So is that ok and should we close this issue now?  Or should we figure out if there should be another means to navigate into the calendar from the input?

**Comment by backwardok** [comment]
I think this never got closed after it was addressed! @monokrome addressed this in https://github.com/airbnb/react-dates/pull/1499
